localization/replace/german/event_localization/lifestyle/warfare/historical_commander_trait_events_l_german.yml

```diff
@@ -1,39 +1,39 @@
 ﻿l_german:
  #Generic Lines
- historical_commander_trait.reject.tt:1 "#F You will look elsewhere for inspiration #!"
+ historical_commander_trait.reject.tt:1 "#F Ihr sucht andernorts nach Inspiration #!"
 
  # Rislav the Righteous
- historical_commander_trait.0001.t:0 ""Rislav the Righteous""
- historical_commander_trait.0001.desc:1 "While studying the tactics of ancient generals I was [ROOT.Char.Custom('surprised_reaction')] to learn about the exploits of Rislav the Righteous against the Alessian Empire:\n\n#F After taking the kingdom back from the Alessian Empire, after his older brother ceded it, Rislav managed to rout the Alessian armies all the way to the Nibenay thanks to agressive harassing tactics and an alliance with the Kingdom of Kvatch. His actions sparked rebellion throughout the Empire, and the Alessian would suffer many more defeats in the next decades.#!"
- historical_commander_trait.0001.a:0 "Rislav annihilated the Alessian armies, and so will I!"
- historical_commander_trait.0001.b:0 "To march from Anvil all the way to the Nibenay requires a great deal of preparation..."
- historical_commander_trait.0001.c:0 "Using his archers to prevent his enemy from catching its breath..."
- historical_commander_trait.0001.d:0 "These tactics might have worked centuries ago, but the world has changed!"
+ historical_commander_trait.0001.t:0 ""Rislav der Rechtschaffene""
+ historical_commander_trait.0001.desc:1 "Beim Studium der Taktiken alter Generäle erfuhr ich [ROOT.Char.Custom('surprised_reaction')] von den Heldentaten Rislavs des Rechtschaffenen gegen das Alessianische Reich: Nachdem er das Königreich vom Alessianischen Reich zurückerobert hatte, nachdem sein älterer Bruder es abgetreten hatte, gelang es Rislav, die alessianischen Armeen dank einer aggressiven Belagerungstaktik und eines Bündnisses mit dem Königreich von Kvatch bis zum Nibenay zu zerschlagen. Seine Taten lösten im gesamten Reich eine Rebellion aus, und die Alessianer sollten in den nächsten Jahrzehnten noch viele weitere Niederlagen erleiden.#!"
```

Comment:
Generäle: Vanilla-üblich wäre "Befehlshaber", weiß aber nicht, wie es sich hier/bei EK verhält.

Letzter Satz: "Seine Taten lösten im gesamten Reich eine Rebellion aus, welche die erste vieler Niederlagen der Alessianer in den folgenden Jahrzehnten sein sollte."